cypress/integration/pages/frontPage/tests.js

```diff
@@ -1,5 +1,7 @@
 // For testing important features that differ between services, e.g. Timestamps.
 // We recommend using inline conditional logic to limit tests to services which differ.
+const serviceHasIndexAlsos = service => service === 'mundo';
```

Comment:
Can we test a site thats live? Like Thai maybe?

Reply:
Updated from `mundo` to `thai`